<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Import Trace</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>721</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>50</height>
         </size>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Trace name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditTrace">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>59</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Heading</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxHeading">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>16</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>59</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>Separator</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxSep">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Whitespace</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Comma</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Semicolon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tab</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>59</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>Escape</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxEscape">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>110</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>\</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&quot;</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>59</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>Quote</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxQuote">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>140</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Simple Quote (')</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Double Quote (&quot;)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Time format</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditTime">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>59</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxID">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>250</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>59</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxTime">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>280</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>310</y>
           <width>59</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxX">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>310</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>340</y>
           <width>59</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxY">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>340</y>
           <width>121</width>
           <height>26</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>367</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Input file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listViewInput"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Data file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableViewOutput"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
